chunked_pooling/chunking.py

```diff
@@ -56,27 +56,38 @@ def __init__(
                 trust_remote_code=True,
             )
         else:
-            self.tokenizer = tokenizer
+            self.tokenizer = tokenizer or AutoTokenizer.from_pretrained("bert-base-uncased")
```

Comment:
I think we should not initialize a bert tokenizer by default, this can lead to unexpected errors